nox/manifest.py

```diff
@@ -164,6 +164,37 @@ def make_session(self, name, func):
     def next(self):
         return self.__next__()
 
+    def notify(self, session):
```

Comment:
You can specify either the name or the parameterized name:

So both of these would work:

  * `notify('unit_tests')`
  * `notify("unit_tests(python_version='3.6')")`